package.json

```diff
@@ -43,17 +43,23 @@
     "clean": "node scripts/clean-modules/index.js",
     "clean:install": "npm run clean && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
+    "compile:heroku": "webpack --config src/config/webpack.prod.config -p",
+    "compile:prod": "webpack --config src/config/webpack.prod.config -p",
     "danger": "danger",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:heroku",
+    "heroku-prebuild": "npm install rimraf",
```

Comment:
Do we need this? I don't think we are compiling anywhere?

Reply:
Maybe not, but i was thinking we might want to delete the 'build' folder on clean eventually.